src/main/java/lotto/model/Win.java

```diff
@@ -0,0 +1,43 @@
+package lotto.model;
+
+import java.math.RoundingMode;
+import java.text.DecimalFormat;
+import java.util.Map;
+
+public enum Win {
+    WIN_3(3, 5000),
+    WIN_4(4, 50000),
+    WIN_5(5, 1500000),
+    WIN_6(6, 2000000000);
```

Comment:
```suggestion
    WIN_6(6, 2_000_000_000);
```
자바에서는 언더바(`_`)를 통해 가독성을 높힐 수 있습니다.